src/Billing/Controllers/StripeController.cs

```diff
@@ -235,14 +240,18 @@ public async Task<IActionResult> PostWebhook([FromQuery] string key)
             }
 
             var subscription = await _stripeFacade.GetSubscription(invoice.SubscriptionId);
+            var customer = await _stripeFacade.GetCustomer(subscription.CustomerId);
```

Comment:
⛏️ Nitpick: should we place this after checking if the feature flag is enabled? Because if its disabled we don't need it, correct?